operator-common/src/main/java/io/strimzi/operator/cluster/operator/resource/cruisecontrol/CruiseControlConfigurationParameters.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.cluster.operator.resource.cruisecontrol;
+
+import java.util.concurrent.TimeUnit;
+
+public enum CruiseControlConfigurationParameters {
+
+    // 'goals' and 'default.goals' are currently not defined, due to unnecessary dependency pulling for now
```

Comment:
We could move the default goal list definitions into this class as well? WDYT @ppatierno?